python/libcasm/mapping/methods/_methods.py

```diff
@@ -0,0 +1,51 @@
+import libcasm.mapping.info as mapinfo
+import libcasm.xtal as xtal
+import numpy as np
+
+
+def map_lattices_without_reorientation(
+    lattice1: xtal.Lattice,
+    lattice2: xtal.Lattice,
+    transformation_matrix_to_super=np.eye(3, dtype=int),
```

Comment:
For documenting optional arrays with a default, I've been [using](https://prisms-center.github.io/CASMcode_pydocs/libcasm/configuration/2.0/reference/libcasm/_autosummary/libcasm.enumerate.ScelEnum.by_volume.html#libcasm.enumerate.ScelEnum.by_volume) the convention
```
    transformation_matrix_to_super: typing.Optional[np.ndarray] = None,
```

With docs:
```
        transformation_matrix_to_super: Optional[np.ndarray] = None,
            An integer shape=(3,3) transformation matrix ...
            If None, :math:`T` is set to the identity matrix.
        
```

And then inside:
```
if transformation_matrix_to_super is None:
    transformation_matrix_to_super = np.eye(3, dtype=int)
```

Otherwise, I think the generated docs print the entire array in function signature.